front/hooks/useAppKeyboardShortcuts.ts

```diff
@@ -0,0 +1,34 @@
+import type { LightWorkspaceType } from "@dust-tt/types";
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+
+export function useAppKeyboardShortcuts(owner: LightWorkspaceType) {
+  const [isNavigationBarOpen, setIsNavigationBarOpen] = useState(true);
+  const router = useRouter();
+
+  useEffect(() => {
+    function handleKeyboardShortcuts(event: KeyboardEvent) {
```

Comment:
`nit` let's use arrow functions when nested within function's code.

Reply:
Why ?

Reply:
Suggested as it helps improve reading/parsing the code when `function` keyword are reserved for top level functions. Anyway, it's a nit!